java/code/src/com/redhat/rhn/common/conf/ConfigDefaults.java

```diff
@@ -369,6 +369,11 @@ public class ConfigDefaults {
      */
     public static final String UNIFY_CUSTOM_CHANNEL_MANAGEMENT = "java.unify_custom_channel_management";
 
+    /**
+     * Specify the number of minutes to wait before performing a system reboot
+     * */
+    public static final String MINUTES_TO_WAIT_BEFORE_REBOOT = "minutes_to_wait_before_reboot";
```

Comment:
Nobody knows that this option exists. I would suggest to do the following:

- use an easier option name and start it with "java." - e.g. java.reboot_wait_time
- specify the default in java/conf/rhn_java.conf and add a comment which explains this option incl. the time unit (minutes)
- take care that the value is ">= 1" - A value of 0 would cause a direct shutdown which makes it impossible for salt to return the result back. You would often have a failed action.
- check the uyuni documentation if reboot is explained and if the 3 minutes is described. If yes, that would be a good point to explain this option. Otherwise the description in the default config file is enough.

Reply:
> use an easier option name and start it with "java."

Why the java prefix?

Reply:
The half is convention. The other reason is, that config values use namespaces. Te java config parser works a bit different than the python. In the python config default files you specify only the var name and the namespace is taken of the filename. When you want to overwrite them in rhn.conf, you have to specify the namespace.
To align this, the namespaces are defined in the java config files directly.
All this still comes from the spacewalk times

Reply:
Ah, I see, thanks!